src/ONE.Models.CSharp/Imposed/Enums/EnumRelativeDateRange.cs

```diff
@@ -9,7 +9,17 @@ public enum EnumRelativeDateRange
         LastYear = 4,
         LastQuarter = 5,
         LastSemiAnnual = 6,
-        Last12Months = 7
-
+        Last12Months = 7,
+        Today = 8,
+        Yesterday = 9,
+        WeekToDate = 10,
+        LastWeek = 11,
+        Last30Days = 12,
+        QuarterToDate = 13,
+        ThisWeek = 14,
+        Last14Days = 15,
+        Last60Days = 16,
+        Last90Days = 17,
+        ThisMonth = 18
```

Comment:
The frontend enum is has more entries than this shared library enum:
![image](https://github.com/AquaticInformatics/ONE.Interfaces.CSharp/assets/1840139/393819eb-7421-4b36-a040-3186c1f22583)